insights/specs_insights_archive.py

```diff
@@ -149,6 +149,9 @@ class InsightsArchiveSpecs(Specs):
     systemctl_list_unit_files = simple_file("insights_commands/systemctl_list-unit-files")
     systemctl_list_units = simple_file("insights_commands/systemctl_list-units")
     systemctl_mariadb = simple_file("insights_commands/systemctl_show_mariadb")
+    systemctl_pulp_workers = simple_file("insights_commands/systemctl_pulp_workers")
+    systemctl_pulp_resmg = simple_file("insights_commands/systemctl_pulp_resmg")
+    systemctl_pulp_celerybeat = simple_file("insights_commands/systemctl_pulp_celerybeat")
```

Comment:
@xiangce modfiy the insights/specs_insights_archive.py, please take a look

Reply:
This should be `insights_commands/systemctl_show_pulp_celerybeat`